src/Lecture2.hs

```diff
@@ -97,7 +113,9 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+
+dropSpaces :: [Char] -> [Char]
+dropSpaces = filter (/= ' ')
```

Comment:
This is a nice and short implementation 👍🏻 

Unfortunately, it doesn't work on infinite lists 😞 

If I generate a list like this: `myList = "foo" ++ repeat ' '` and do `take 5 (dropSpaces myList)`, the function will hang. However, it's possible to implement this function so the above won't hang.

Reply:
Thanks. I have fixed it.